#controller_joint_names: ['', 'joint01', 'joint02', 'joint03', 'joint04', 'joint05', 'joint06', 'joint07', 'joint08', 'joint09', 'joint10', 'joint11', ]

joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50   

  # Effort Controllers ---------------------------------------
joint01_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint01
    pid: {p: 0, i: 0, d: 0}
    
joint02_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint02
    pid: {p: 0, i: 0, d: 0}
    
joint03_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint03
    pid: {p: 0, i: 0, d: 0}
    
joint04_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint04
    pid: {p: 0, i: 0, d: 0}
    
joint05_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint05
    pid: {p: 0, i: 0, d: 0}
    
joint06_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint06
    pid: {p: 0, i: 0, d: 0}
    
joint07_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint07
    pid: {p: 0, i: 0, d: 0}
    
joint08_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint08
    pid: {p: 0, i: 0, d: 0}
    
    
joint09_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint09
    pid: {p: 0, i: 0, d: 0}
    
    
joint10_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint10
    pid: {p: 0, i: 0, d: 0}
    
joint11_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint11
    pid: {p: 0, i: 0, d: 0}
